install.sh

```diff
@@ -154,6 +155,42 @@ __atuin_install_cargo(){
 	cargo install atuin
 }
 
+__atuin_install_binary() {
+    local target
+    local arch=$(uname -sm)
+
+    case $arch in
+    "Linux x86_64")
+        target="x86_64-unknown-linux-musl"
+        ;;
+    "Linux aarch64")
+        target="aarch64-unknown-linux-gnu"
+        ;;
+    "Darwin x86_64")
+        target="x86_64-apple-darwin"
+        ;;
+    "Darwin arm64")
+        target="aarch64-apple-darwin"
+        ;;
+    *)
+        __atuin_install_unsupported
+        return
+        ;;
+    esac
+
+    local latest_version=$(curl -s https://api.github.com/repos/atuinsh/atuin/releases/latest | grep tag_name  | grep -Eo "v[0-9]+\.[0-9]+\.[0-9]+")
+    local archive_name=atuin-$latest_version-$target
+    local binary_url="https://github.com/atuinsh/atuin/releases/download/v17.0.1/$archive_name.tar.gz"
```

Comment:
This hardcodes v17.0.1

Reply:
Made a few updates for the review, shellcheck and a small fix for darwin's `stat` utility.

Reply:
@ellie , did you get a chance to see the changes?